docs/DeveloperGuide.md

```diff
@@ -234,6 +234,50 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 _{more aspects and alternatives to be added}_
 
+### Tag adding feature
+
+### Implementation
+
+The tag adding mechanism is facilitated bt `TagAddCommand` and `TagAddCommandParser`.
+Additionally, The mechanism utilises the following operations in `UniqueTagList` and `UniquePersonList`.
+
+* `boolean hasTag(Tag tag)` - Checks if the tagList has the tag.
```

Comment:
Both classes contains this method? If not, specify the class with #

Reply:
I made the changes.